releases_test.go

```diff
@@ -141,30 +141,33 @@ func Test_Releases(t *testing.T) {
 			}
 
 			crd := v1alpha1.NewReleaseCRD()
-			var v apiextensions.CustomResourceValidation
-			// Convert the CRD validation into the version-independent form.
-			err = v1beta1.Convert_v1beta1_CustomResourceValidation_To_apiextensions_CustomResourceValidation(crd.Spec.Validation, &v, nil)
-			if err != nil {
-				t.Fatal(microerror.Mask(err))
-			}
 
-			validator, _, err := validation.NewSchemaValidator(&v)
-			if err != nil {
-				t.Fatal(microerror.Mask(err))
-			}
+			for _, crdVersion := range crd.Spec.Versions {
```

Comment:
Should this iterate through or just take the highest version in there?

Reply:
Tbh I'm not sure, I just adapted the tests in the same way that we did it in [apiextensions](https://github.com/giantswarm/apiextensions/commit/1c32c5de067017dc8fe52dc718543b8b5e5e7179). 

Note that those are crd versions (so this should be just v1 in our case I think?) and not release versions.

Reply:
In this case, these are versions of the Releases API defined by the CRD rather than the version of the CRD itself (v1/v1beta1). It's possible to have different OpenAPIV3 schemata for different versions which is why I added this loop. Currently we only have one version of Releases so it's not that important either way.